src/implementation/Client/GRPCClient/configuration.ts

```diff
@@ -135,21 +127,16 @@ export default class GRPCClientConfiguration implements IClientConfiguration {
 
     stream.on("data", async (data: SubscribeConfigurationResponse) => {
       streamId = data.getId();
+      const items = data.getItemsMap();
+
+      if (items.getLength() == 0) {
```

Comment:
FYI, I added this in my recent commit. Apparently `stream.on` is returning two entries per change. The first one is empty, and second one contains actual data. Added this snippet to simply ignore the first empty callback.